iwyu_path_util.h

```diff
@@ -66,14 +66,22 @@ string MakeAbsolutePath(const string& base_path, const string& relative_path);
 // Get the parent of path.
 string GetParentPath(const string& path);
 
+// Try to strip the prefix_path from the front of path.
+// path assumed to be normalized but either absolute or relative
```

Comment:
Maybe `// The path argument is assumed [...] relative.`

Full sentence, trailing period.